src/components/Voucher/Voucher.js

```diff
@@ -0,0 +1,86 @@
+
+import React from 'react';
+
+
+import styles from './Voucher.module.css'
+
+function Voucher( {children, price, carrier, originFrom, destinationFrom, timeStartFrom, timeEndFrom, dateFrom, durationFrom, stopsFrom, originTo, destinationTo, dateTo, stopsTo, durationTo, timeStartTo, timeEndTo}) {
+    console.log(stopsFrom);
+    let stopsCities1 = stopsFrom.join(", ");
+    let stopsCities2 = stopsTo.join(", ");
+
+    function convertPrice(price)
+    {
+        let p = price % 1000;
+        if(p < 10) return "00" + p;
+        if(p < 100) return "0" + p;
+        return p;
+    }
+
+    return (
+    <div className="voucher">
+    <div className={styles.first}>
+      <div className={styles.voucher__price}>
+  {`${Math.floor(price / 1000)} ${convertPrice(price)}`} Р
```

Comment:
It could be showed easier.
```suggestion
  {Math.floor(price / 1000)} {convertPrice(price)} Р
```

Also all formating logic should be placed in one function.